### ISSUE TITLE
Version 2.2.0 does not produce any predictions anymore for netmhcpan class 1

### ISSUE THREAD

**Issue by steffenlem** [nf-core/epitopeprediction#195]
### Description of the bug

When running the pipeline using version 2.2.0 no predictions are performed by netmhcpan 4.1 for class 1 peptides. However, the pipeline runs through without any errors, just no predictions are produced. Version 2.1.0 works fine.

### Command used and terminal output

```console
nextflow run nf-core/epitopeprediction -r 2.2.0 \
        --input '/path/to/input_sheet.csv' \
        -profile cfc \
        --outdir out \
        --tools netmhcpan-4.1 \
        --netmhcpan_path '/path/to/netMHCpan-4.1b.Linux.tar.gz'
```

### Relevant files

Contains:
- epitope_prediction_in.tsv
- sample_1_prediction_report.json

[files.zip](https://github.com/nf-core/epitopeprediction/files/10955185/files.zip)

**Comment by jonasscheid** [comment]
Error is reproducible. Syfpeithi works. Seems to be related to external tools.
Maybe due to the changes [here](https://github.com/nf-core/epitopeprediction/commit/93df79b17838e86ef596bbaec9a8ba1b804cdb59), [here](https://github.com/nf-core/epitopeprediction/commit/db9ec039602c312935f3959f3e33f66297f7f200) or [here](https://github.com/nf-core/epitopeprediction/commit/16e83a7758d70d617f7235451c8e16aeac228f15) @ggabernet

**Comment by ggabernet** [comment]
Hi, thanks for reporting the error. I think I found the source and it will be solved in PR #196

I also made the prediction output not optional, as it is important to be able to identify these errors for a start in the CI tests.

**Comment by ggabernet** [comment]
Hello @steffenlem, could you try if this is solved in `ggabernet/epitopeprediction -r dev` ?
